google-account-plugin/src/com/google/cloud/tools/intellij/login/ui/GoogleLoginUsersPanel.java

```diff
@@ -186,11 +187,9 @@ public void mouseDragged(MouseEvent event) {
     boolean noUsersAvailable =
         (listModel.getSize() == 1) && (listModel.get(0) instanceof NoUsersListItem);
     JButton addAccountButton = new JButton(noUsersAvailable ? SIGN_IN : ADD_ACCOUNT);
-    addAccountButton.addActionListener(new ActionListener() {
-      @Override
-      public void actionPerformed(ActionEvent event) {
-        Services.getLoginService().logIn();
-      }
+    addAccountButton.addActionListener(event -> {
+      SwingUtilities.getWindowAncestor(this).dispose();
```

Comment:
> I also included a little fix to close the sign-in popup when transitioning to the browser so it doesn't float over top of the browser endlessly until you go back to IntelliJ and remove focus from it.

Is this the fix? If not could you point us to it? (sorry if you did already and I missed it)